### PR TITLE
seeded mysql db

### PR DESCRIPTION
**Pull request by RayDunningII**

### CODE REVIEW

File: server/express.js [hunk 1]
```diff
@@ -1,16 +1,21 @@
 const express = require('express');
 const graphqlHTTP = require('express-graphql');
 const { buildSchema } = require('graphql');
+require('.envDOT')
```
Comment (zachystuff) [inline server/express.js:4]:
```suggestion
require('dotenv').config()
```

File: database/schema.sql [hunk 2]
```diff
@@ -0,0 +1,49 @@
+DROP DATABASE IF EXISTS mvps;
```
Comment (zachystuff) [inline database/schema.sql:1]: The database should be seeded with knexjs. refer to knew js documentation. not MySQL.

File: .env [hunk 3]
```diff
@@ -0,0 +1,6 @@
+DB_HOST=127.0.0.1
```
Comment (zachystuff) [inline .env:1]: .env file should be added to gitignore file. shouldn't be uploaded.

File: server/express.js (hunk 1 above)
Comment (RayDunningII) [reply server/express.js:4]: Oh yeah. I just threw that seed together. I guess we gotta get that seeded with kinex tomorrow

**Comment by RayDunningII** [comment]
Please judge

**Pull request closed without merging**